mysql fail to boot

```
2026-04-25T09:44:24.589782Z 0 [ERROR] [MY-000068] [Server] unknown option '--skip-host-cache'.

2026-04-25T09:44:24.590532Z 0 [ERROR] [MY-010119] [Server] Aborting

2026-04-25T09:44:26.022829Z 0 [System] [MY-010910] [Server] /usr/sbin/mysqld: Shutdown complete (mysqld 8.4.8)  MySQL Community Server - GPL.

2026-04-25T09:44:26.022909Z 0 [System] [MY-015016] [Server] MySQL Server - end.
```

--- a/_dev/environment/databases/mysql/conf/my.cnf
+++ b/_dev/environment/databases/mysql/conf/my.cnf
@@ -22,7 +22,6 @@
 # this will increase compatibility with older clients. For background, see:
 # https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin
 # default-authentication-plugin=mysql_native_password
-skip-host-cache
 skip-name-resolve
 datadir=/var/lib/mysql
 socket=/var/run/mysqld/mysqld.sock

